### PR TITLE
update members.md added my info

### PR DESCRIPTION
**Pull request by bagladivyang03**

### CODE REVIEW

**Comment by bagladivyang03** [comment]
@ahammadshawki8 could u pls review and merge it :)

File: Members.md [hunk 1]
```diff
@@ -199,3 +199,10 @@ __Tech Stack__:  C/C++, Java, Kotlin, Firebase
 __Social Handles__:  [LinkedIn](https://www.linkedin.com/in/rahulsain/)  | [GitHub](https://github.com/rahulsain) | [Twitter](https://twitter.com/rahul_sain01) | [Instagram](instagram.com/_rahulsain/)  
 __Website__:  [Podcast](https://anchor.fm/rahulsain) | [Listen to my Podcast on Spotify](https://open.spotify.com/show/1CJoDKXx9tmk5BxI0pvgGX)  
 
+
+- __Name__:  Divyang Bagla 
+__Country__:  India  
+__About__:  I am a Junior year CS undergraduate from MIT-WPU,Pune.I know web develeopment technologies like django, Bootstrap. I like to do the competitive programming.I love learning the new things. 
+__Tech Stack__:  C/C++, Python,MySQL,Django, BootStrap 
+__Social Handles__:  [LinkedIn](https://www.linkedin.com/in/divyang-bagla-976233176/)  | [GitHub](https://github.com/bagladivyang03)  
+__Website__:  [Portfolio](https://divyangbagla.vercel.app/)
```
Comment (ahammadshawki8) [inline Members.md:208]: Good job 👏

```suggestion

- __Name__: Divyang Bagla 
__Country__: India  
__About__: I am a Junior year CS undergraduate from MIT-WPU, Pune.I know web development technologies like Django, Bootstrap. I like to do competitive programming. I love learning new things. 
__Tech Stack__: C/C++, Python,MySQL,Django, BootStrap 
__Social Handles__: [LinkedIn](https://www.linkedin.com/in/divyang-bagla-976233176/) | [GitHub](https://github.com/bagladivyang03)  
__Website__: [Portfolio](https://divyangbagla.vercel.app/)  
```

**Review by ahammadshawki8 — CHANGES_REQUESTED** [review]
Please solve this problems

File: Members.md (hunk 1 above)
Comment (bagladivyang03) [reply Members.md:208]: Thanks @ahammadshawki8 :)

**Review by plazzy99 — APPROVED** [review]

**Pull request merged**